spec/models/disk.rb

```diff
@@ -4,4 +4,24 @@ class Disk < ActiveRecord::Base
   include ArchivedMixin
 
   belongs_to :hardware
+
+  # A performance improvement was introduced in Rails 6:
+  #
+  #   https://github.com/rails/rails/commit/cc2d614e
+  #
+  # Causes the `present` column in this class to raise the following error:
+  #
+  #   ActiveRecord::DangerousAttributeError:
+  #     present? is defined by Active Record. Check to make sure that you don't
+  #     have an attribute or method with the same name.
+  #
+  # Since there is no whitelist for this method, this attempts to circumvent
+  # that autogenerated error to allow our previously named column to still work
+  # properly.
+  #
+  def self.dangerous_attribute_method?(name)
```

Comment:
@NickLaMuro could we just change the name of the column in the `spec/schema.rb`?  The schema is a clone of MIQ for testing so the specific column name doesn't matter.

For MIQ proper thinking we could rename this column to `:connected` which is actually a more accurate name

Reply:
Switched to `:connected` as suggested.